packages/editor-ui/src/components/JsEditor/JsEditor.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+	<div ref="jsEditor" class="ph-no-capture js-editor"></div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { acceptCompletion, autocompletion } from '@codemirror/autocomplete';
+import { indentWithTab, history, redo, toggleComment } from '@codemirror/commands';
+import { foldGutter, indentOnInput } from '@codemirror/language';
+import { javascript } from '@codemirror/lang-javascript';
+import { lintGutter } from '@codemirror/lint';
+import type { Extension } from '@codemirror/state';
+import { EditorState } from '@codemirror/state';
+import type { ViewUpdate } from '@codemirror/view';
+import {
+	dropCursor,
+	EditorView,
+	highlightActiveLine,
+	highlightActiveLineGutter,
+	keymap,
+	lineNumbers,
+} from '@codemirror/view';
+
+import { codeNodeEditorTheme } from '../CodeNodeEditor/theme';
+
+export default defineComponent({
+	name: 'js-editor',
+	props: {
+		modelValue: {
+			type: String,
+			required: true,
+		},
+		isReadOnly: {
+			type: Boolean,
+			default: false,
+		},
+		rows: {
+			type: Number,
+			default: -1,
```

Comment:
Isn't this going to break the editor if user doesn't provide `rows`? 

In `CodeNodeEditor/theme.ts` the row count is used as `customMinHeight` in `{ minHeight: customMinHeight ? `${Number(customMinHeight) * 1.3}em` : '10em' })`.

I'd say default should be `1` instead of `-1`, for all editors.